### ISSUE TITLE
ERROR: Transactions outside US

### ISSUE THREAD

**Issue by opn7d** [umkc-cs-451-2020-spring/semester-project-group-5#69]
Transactions won't log correctly if a purchase/s is made outside of the United States of America.

**Comment by mikelhomeless** [comment]
NICE

**Comment by mikelhomeless** [comment]
<img width="115" alt="Screen Shot 2020-04-27 at 5 29 45 PM" src="https://user-images.githubusercontent.com/25778778/80427115-b0c9f780-88ac-11ea-9a54-291d60eed58d.png">
NICE

**Comment by mikelhomeless** [comment]
I agree that this is an issue with our transactions model. When we wrap up the UI, we should look into this and determine if we should move forward with supporting locations outside the continental U.S.
